quiz.js

```diff
@@ -8,6 +8,18 @@
 
 /* your code starts here */
 
+function square(x) {
+  return x*x
+}
+
+function cube(x) {
+  return x*x*x
```

Comment:
your solution is technically correct, however the instructions told you to use square in your cube and quad functions